### DISCUSSION TITLE
Translation into Spanish

### DISCUSSION THREAD

**Discussion by ImEcuadorian** [palexdev/MaterialFX#258] [Archived]
There is translation into spanish

[mfxlang_es.zip](https://github.com/palexdev/MaterialFX/files/10096932/mfxlang_es.zip)

![Screenshot 2022-11-19 225910](https://user-images.githubusercontent.com/46403054/202884715-14d68745-8d6e-4b85-befb-c2580dee52c1.png)
You need to create a new .properties like in photo and then add a new language on enum
![Screenshot 2022-11-19 225940](https://user-images.githubusercontent.com/46403054/202884716-463bbccd-170b-4584-bccc-463e3d12368c.png)

**Comment by palexdev** [comment]
Can't you send a PR? It is the proper way to do this

**Reply by ImEcuadorian** [reply]
Yeah, sorry for that but I send a zip with .properties
